odinmain/src/main/kotlin/me/odinmain/features/impl/floor7/p3/termsim/TermSimGui.kt

```diff
@@ -117,4 +118,6 @@ fun openRandomTerminal(ping: Long = 0L, const: Long = 0L) {
         TerminalTypes.MELODY -> {}
         TerminalTypes.NONE -> {}
     }
-}
\ No newline at end of file
+}
+
+fun isTermSimOpen() = OdinMain.mc.currentScreen is TermSimGui
```

Comment:
Mod code style wise we don't make functions for a single check please implement the check instead of a function for it
Also make sure to import me.odinmain.OdinMain.mc instead of me.odinmain.OdinMain